linux-tools/traceroute/traceroute.sh

```diff
@@ -88,6 +88,7 @@ function run_test()
 	# If this is the only test that passes for a system, then we can guess
 	# that there is some system in the probe path that expects some delay
 	# between probes
+	sleep 05
```

Comment:
In general, if you can add a waiting function for a certain system condition, that's always better than a flat sleep. Can you do that in this case?